core/domain/android_validation_domain.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Perhaps rename file to android_validation_constants.py since these aren't "real" domain objects

Reply:
Done